glossary.html

```diff
@@ -15,7 +15,7 @@
 </head>
 
 <body>
-    <div id=width-wrap>
+    <div class="width-wrap">
```

Comment:
Incorrect attribute name.
It's considered best practice to always quote your attribute values